### PR TITLE
Fix to don't update checkpoint if height has not advanced

### PR DESCRIPTION
**Pull request by bluele**

### CODE REVIEW

File: pkg/relay/ethereum/chain.go [hunk 1]
```diff
@@ -383,6 +387,10 @@ func (c *Chain) QueryUnfinalizedRelayAcknowledgements(ctx core.QueryContext, cou
 		return nil, err
 	}
 
+	if checkpoint > ctx.Height().GetRevisionHeight() {
+		logger.Info("checkpoint is greater than target height", "checkpoint", checkpoint, "height", ctx.Height().GetRevisionHeight())
```
Comment (siburu) [inline pkg/relay/ethereum/chain.go:391]: It is more informative to log `RECEIVE checkpoint is ...`.

File: pkg/relay/ethereum/chain.go [hunk 2]
```diff
@@ -324,6 +324,10 @@ func (c *Chain) QueryUnfinalizedRelayPackets(ctx core.QueryContext, counterparty
...
```
Comment (siburu) [inline pkg/relay/ethereum/chain.go:328]: It is more informative to log `SEND checkpoint is ...`.

**Review by siburu — COMMENTED** [review]
@bluele Thank you. I reviewed your code and requested small changes.

**Comment by bluele** [comment]
@siburu I have applied your suggestions, can you review again?

**Review by siburu — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**